src/tools/load/load.cpp

```diff
@@ -0,0 +1,90 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+--*/
+
+#include <stdio.h>
+#include "quic_datapath.h"
+#include "msquic.hpp"
+
+const MsQuicApi* MsQuic;
+volatile long ConnectedCount;
+volatile long ConnectionsActive;
+
+void ResolveServerAddress(const char* ServerName, QUIC_ADDR& ServerAddress) {
+    CxPlatSystemLoad();
+    CxPlatInitialize();
+    CXPLAT_DATAPATH* Datapath = nullptr;
+    //QuicAddrSetFamily(&ServerAddress, AF_INET);
```

Comment:
not needed anymore?

Reply:
I'm leaving it there for manual usage in the future, but I don't want it in by default.